front/app/containers/Admin/formBuilder/components/FormBuilderSettings/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from 'react';
+
+// styles
+import { colors } from 'utils/styleUtils';
+
+// components
+import {
+  Box,
+  Toggle,
+  Text,
+  stylingConsts,
+} from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+import InputMultilocWithLocaleSwitcher from 'components/UI/InputMultilocWithLocaleSwitcher';
+import { SectionField, SectionTitle } from 'components/admin/Section';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+// intl
+import messages from '../messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+// Types
+import { Multiloc } from 'typings';
+import {
+  IFlatCustomField,
+  IFlatUpdateCustomField,
+} from 'services/surveyCustomFields';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+interface Props {
+  field?: IFlatCustomField;
+  onDelete: (fieldId: string) => void;
+  onFieldChange: (field: IFlatUpdateCustomField) => void;
+  onClose: () => void;
+}
+
+const SurveyBuilderSettings = ({
+  field,
+  onDelete,
+  onFieldChange,
+  onClose,
+}: Props) => {
+  // I'm keeping this form as simple as possible using state pending form rework from (TEC-35)
```

Comment:
I think it would be better to add TODO to the comment, so that we can find it easily together with other TODOs in the code.